src/shared/ntdef.rs

```diff
@@ -5,22 +5,27 @@
 // All files in the project carrying such notice may not be copied, modified, or distributed
 // except according to those terms.
 //! Type definitions for the basic types.
+use core::{mem, ptr};
```

Comment:
What if you imported these items directly inside the function where you used them?